addons/sale/views/sale_order_views.xml

```diff
@@ -668,58 +640,47 @@
                                             class="btn-secondary"/>
                                 </control>
                                 <templates>
-                                    <t t-name="kanban-box">
-                                        <div t-attf-class="oe_kanban_card oe_kanban_global_click ps-0 pe-0 {{ record.display_type.raw_value ? 'o_is_' + record.display_type.raw_value : '' }}">
-                                            <t t-if="!record.display_type.raw_value">
-                                                <div class="row g-0">
-                                                    <div class="col-2 p-1">
-                                                        <img t-att-src="kanban_image('product.product', 'image_128', record.product_id.raw_value)" t-att-title="record.product_id.value" t-att-alt="record.product_id.value" style="max-width: 100%;"/>
+                                    <t t-name="kanban-card" class="row g-0 ps-0 pe-0">
+                                        <t t-if="!record.display_type.raw_value">
+                                            <aside class="col-2 p-1">
+                                                <span t-att-title="record.product_id.value">
+                                                    <field name="product_id" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain w-100'}" t-att-alt="record.product_id.value"/>
```

Comment:
t-att- doesn't work on field nodes. But you can just remove it, the correct alt will be set automatically to the display name of the many2one